src/tests/Microsoft.PowerFx.Interpreter.Tests/FunctionCompilationTests.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using Microsoft.PowerFx.Core.Public.Types;
+using Microsoft.PowerFx.Core.Types;
+using Microsoft.PowerFx.Interpreter;
+using Xunit;
+
+namespace Microsoft.PowerFx.Tests
+{
+    public class FunctionCompilationTests
+    {
+        [Theory]
+        [InlineData("Switch(A, 2, \"two\", \"other\")")]
```

Comment:
.txt tests instead?

Reply:
Cannot do it now - IfError doesn't expose the 'FirstError' scope variable yet

Reply:
Indeed, that will be added in a future PR